### PR TITLE
 Modify Serial_SINGLE_MEASUREMENT, created app.txt, example.md, ardui…

### PR DESCRIPTION
**Pull request by Angchau**
Added arduino.ino and docs (app.txt example.md etc)

Modified the type of Datacontainer from plotly to "ordered pair" 

Explained how to use the node with example.md

### CODE REVIEW

File: INSTRUMENTS/SERIAL/SERIAL_SINGLE_MEASUREMENT/SERIAL_SINGLE_MEASUREMENT.py [hunk 1]
```diff
@@ -29,8 +29,10 @@ def SERIAL_SINGLE_MEASUREMENT(dc_inputs, params):
     print(",")
     println(reading1)
 
-    If there is more than one column, the SELECT_ARRAY node must be
-    used after this node.
+    This node will receive the data from the arduino as a string and will split
```
Comment (smahmed776) [inline INSTRUMENTS/SERIAL/SERIAL_SINGLE_MEASUREMENT/SERIAL_SINGLE_MEASUREMENT.py:32]: Can we follow [styleguide](https://github.com/flojoy-io/nodes/blob/main/node_styleguide.md) for docstring?

File: INSTRUMENTS/SERIAL/SERIAL_SINGLE_MEASUREMENT/SERIAL_SINGLE_MEASUREMENT.py [hunk 2]
```diff
@@ -44,19 +46,19 @@ def SERIAL_SINGLE_MEASUREMENT(dc_inputs, params):
     # The first reading is commonly empty.
     s = ser.readline().decode()
 
-    # Some readings may be empty. Try a second time if so.
+    # Some readings may be empty at first because it takes time before receiving the data. Try a second time if so.
```
Comment (smahmed776) [inline INSTRUMENTS/SERIAL/SERIAL_SINGLE_MEASUREMENT/SERIAL_SINGLE_MEASUREMENT.py:49]: How about doing it with a while loop as follows:
```
s = ""
while s == "":
        s = ser.readline().decode()
 reading = s[:-2].split(",")
 // rest of the code
```

**Review by smahmed776 — CHANGES_REQUESTED** [review]
Left couple of comments, looks good otherwise.

File: INSTRUMENTS/SERIAL/SERIAL_SINGLE_MEASUREMENT/SERIAL_SINGLE_MEASUREMENT.py (hunk 2 above)
Comment (Angchau) [reply INSTRUMENTS/SERIAL/SERIAL_SINGLE_MEASUREMENT/SERIAL_SINGLE_MEASUREMENT.py:49]: Yes definitely !!

File: INSTRUMENTS/SERIAL/SERIAL_SINGLE_MEASUREMENT/SERIAL_SINGLE_MEASUREMENT.py [hunk 3]
```diff
@@ -29,34 +30,34 @@ def SERIAL_SINGLE_MEASUREMENT(dc_inputs, params):
...
+
+    Parameters :
+    ------------
+    BAUD_RATE: Float
+    Baud rate for the serial communication.
 
-    params:
-    BAUD_RATE: Baud rate for the serial device.
-    com_port: COM port of the serial device
+    COM_PORT: String
```
Comment (smahmed776) [inline INSTRUMENTS/SERIAL/SERIAL_SINGLE_MEASUREMENT/SERIAL_SINGLE_MEASUREMENT.py:42]: Could you please match parameter names from manifest file?

File: INSTRUMENTS/SERIAL/SERIAL_TIMESERIES/SERIAL_TIMESERIES.py [hunk 4]
```diff
@@ -41,10 +41,11 @@ def SERIAL_TIMESERIES(dc_inputs, params):
     num_readings * record_period is roughly the run length in seconds.
```
Comment (smahmed776) [inline INSTRUMENTS/SERIAL/SERIAL_TIMESERIES/SERIAL_TIMESERIES.py:41]: Could you please make sure that all parameter names match with the manifest file?

**Pull request merged**